documents/payment.md

```diff
@@ -863,6 +863,253 @@ Razorpay::Iin.fetch(tokenIin);
+|-------------|---------|---------------------------------------------------------------------|
+| paymentId*  | integer | Unique identifier of the payment                                    |
+| expand[]    | string  | Use to expand the card details when the payment method is `card`.   |
+
+**Response:** <br>
+
+```json
+{
+  "id": "pay_H9oR0gLCaVlV6m",
+  "entity": "payment",
+  "amount": 100,
+  "currency": "INR",
+  "status": "failed",
+  "order_id": "order_H9o58N6qmLYQKC",
+  "invoice_id": null,
+  "terminal_id": "term_G5kJnYM9GhhLYT",
+  "international": false,
+  "method": "card",
+  "amount_refunded": 0,
+  "refund_status": null,
+  "captured": false,
+  "description": null,
+  "card_id": "card_H9oR0ocen1cmZq",
+  "card": {
+    "id": "card_H9oR0ocen1cmZq",
+    "entity": "card",
+    "name": "Gaurav",
+    "last4": "1213",
+    "network": "RuPay",
+    "type": "credit",
+    "issuer": "UTIB",
+    "international": false,
+    "emi": false,
+    "sub_type": "business"
+  },
+  "bank": null,
+  "wallet": null,
+  "vpa": null,
+  "email": "gaurav.kumar@example.com",
+  "contact": "+919000090000",
+  "notes": {
+    "email": "gaurav.kumar@example.com",
+    "phone": "09000090000"
+  },
+  "fee": null,
+  "tax": null,
+  "error_code": "BAD_REQUEST_ERROR",
+  "error_description": "Card issuer is invalid",
+  "error_source": "customer",
+  "error_step": "payment_authentication",
+  "error_reason": "incorrect_card_details",
+  "acquirer_data": {
+    "auth_code": null,
+    "authentication_reference_number": "100222021120200000000742753928"
+  },
+  "created_at": 1620807547
+}
+```
+
+-------------------------------------------------------------------------------------------------------
+
+### Fetch a Payment (With Expanded Offers Details)
+
+```rb
+Razorpay::Payment.fetch("pay_XXXXXXXXXXXXXX").expendDetails({"expand[]": "offers"})
+```
+
+**Parameters:**
+
+| Name        | Type    | Description                                                    |
+|-------------|---------|----------------------------------------------------------------|
+| paymentId*  | integer | Unique identifier of the payment                               |
+| expand[]    | string  | Use to expand the emi details when the payment method is emi.  |
+
+**Response:** <br>
+
+```json
+{
+  "id": "pay_DG4ZdRK8ZnXC3k",
+  "entity": "payment",
+  "amount": 200000,
+  "currency": "INR",
+  "status": "authorized",
+  "order_id": null,
+  "invoice_id": null,
+  "international": false,
+  "method": "emi",
+  "amount_refunded": 0,
+  "refund_status": null,
+  "captured": false,
+  "description": null,
+  "card_id": "card_DG4ZdUO3xABb20",
+  "bank": "ICIC",
+  "wallet": null,
+  "vpa": null,
+  "email": "gaurav@example.com",
+  "contact": "+919972000005",
+  "notes": [],
+  "fee": null,
+  "tax": null,
+  "error_code": null,
+  "error_description": null,
+  "error_source": null,
+  "error_step": null,
+  "error_reason": null,
+  "emi": {
```

Comment:
This response seems to contain expanded details for emi. Can you update it relevant for offers ?

Reply:
Updated to offer response